star.wdl

```diff
@@ -102,6 +102,7 @@ task Star {
 
         Int runThreadN = 4
         String memory = "48G"
```

Comment:
I have been running star lately. The memory usage is roughly equal (it seems to be slightly lower based on MaxRSS) than the index size. May we can do `ceil(size(indexFiles, "G") * 1.2)`  as a memory estimate?